tests/samples/test_api.py

```diff
@@ -105,31 +105,7 @@ async def get_sample_data(mongo, fake2, pg, static_time):
     return user.id
 
 
-@pytest.mark.apitest
-@pytest.mark.parametrize(
-    "find,per_page,page,labels",
-    [
-        (None, None, None, None),
-        (None, 2, 1, None),
-        (None, 2, 2, None),
-        ("gv", None, None, None),
-        ("sp", None, None, None),
-        (None, None, None, [3]),
-        (None, None, None, [2, 3]),
-        (None, None, None, [0]),
-    ],
-)
-async def test_find(
-    find,
-    per_page,
-    page,
-    labels,
-    snapshot,
-    fake2,
-    spawn_client,
-    static_time,
-    pg: AsyncEngine,
-):
+async def setup_initial_test_setup(fake2, spawn_client, static_time):
```

Comment:
Name is confusing, would change to something like: "setup_find_user_test" as it reduces redundancy as makes it clear that the function is for setting up one particular set of tests

That said, since we are in a class the invocation of the initial test setup can be moved into a setup_method which automates running the setup before every test is run

https://doc.pytest.org/en/latest/how-to/xunit_setup.html